File: Domain/Quiz.cs
using System;
using System.Collections.Generic;

namespace Domain
{
    public class Quiz
    {
        public int Id {get;set;}

        public string Title { get; set; }

        public string Description {get;set;}

        public string Picture {get; set;}
        public ICollection<Question> Questions{get;set;}= new List<Question>();

       
        
        
    }
}

File: client-app/src/app/models/quiz.ts
export interface Question {
        id: number;
        questionDescription: string;
        option1: string;
        answer1: boolean;
        option2: string;
        answer2: boolean;
        option3: string;
        answer3: boolean;
        option4: string;
        answer4: boolean;
    }
export interface Quiz {
        id: number;
        title: string;
        description: string;
        picture: string;
        questions?: Question[];
    }
    export interface Category {
        id: number;
        name: string;
        quizzes: Quiz[];
    }


